build_docs.py

```diff
@@ -48,6 +64,31 @@
 import tomlkit
 import urllib3
 import zc.lockfile
+from platformdirs import user_config_path, site_config_path
+
+ENV_CONF_FILE = None
+_user_config_path = user_config_path("docsbuild-scripts")
+_site_config_path = site_config_path("docsbuild-scripts")
+if _user_config_path.is_file():
+    ENV_CONF_FILE = _user_config_path
+elif _site_config_path.is_file():
+    ENV_CONF_FILE = _site_config_path
+
+if ENV_CONF_FILE:
+    print(f"Reading environment variables from {ENV_CONF_FILE}")
+    if ENV_CONF_FILE == _site_config_path:
+        print(f"You can override settings in {_user_config_path}")
+    elif _site_config_path.is_file():
+        print(f"Overriding {_site_config_path}")
+    with open(ENV_CONF_FILE, "r") as f:
+        for key, value in tomlkit.parse(f.read()).get("env", {}).items():
+            print(f"Setting {key} in environment")
```

Comment:
Is it worth using `logging` instead of `print`? If so, we'd need to configure env vars after logging has been set up.

Reply:
https://github.com/python/docsbuild-scripts/pull/269/commits/cd081deb16fa1a41d045559dc49d28a773009522